pages/index.js

```diff
@@ -19,9 +22,7 @@ export default function HomePage({
               activity={activity}
               onToggleFavorite={onToggleFavorite}
               isFavorite={
-                favoriteActivitiesList.find(
-                  (favActivity) => favActivity._id === activity._id
-                )?.isFavorite
+                session ? userData[0]?.favorites.includes(activity._id) : false
```

Comment:
```suggestion
                session ? userData[0]?.favorites.includes(activity._id) 
```

Reply:
Since this is a ternary-operator, we need ": false" here, don't we?